library/mediawiki

```diff
@@ -2,7 +2,7 @@ Maintainers: David Barratt <dbarratt@wikimedia.org> (@davidbarratt),
              Kunal Mehta <legoktm@wikimedia.org> (@legoktm),
              addshore <addshorewiki@gmail.com> (@addshore)
 GitRepo: https://github.com/wikimedia/mediawiki-docker.git
-GitCommit: 38d4e7f77f8b5b4efda91f969ccba2475fc2c597
+GitCommit: dc240fc8771ce74bc2c420eb1283d718ebf1abe0
 Architectures: amd64, arm32v5, arm32v7, arm64v8, i386, ppc64le
 
 Tags: 1.35.1, 1.35, lts, stable, latest
```

Comment:
```suggestion
Tags: 1.35.2, 1.35, lts, stable, latest
```

Looks like this is missing all the image tag bumps (here and below).

Reply:
Thanks, this is my first time doing an image update like this. Fixed!